src/main/java/dev/yumi/gradle/licenser/api/comment/XmlStyleHeaderComment.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2023 Yumi Project
```

Comment:
That's weird, did you copy/paste files or did you run the applyLicenses task after writing the files?
Those new files definitely should be dated to 2024.